From fbe0bc808b3dfff80b8743003f5be5df7d7eaa93 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 14 Mar 2019 03:02:57 -0700
Subject: [PATCH 0561/1851] dt-bindings: media: xilinx: tpg: Update for v8.0

commit 6a3a55e93df710579e5dfb2335c576844466dd5b from
https://github.com/Xilinx/linux-xlnx.git

Update the bindings for v8.0 of IP by introducing new compatible string.
Make xlnx,max-width and xlnx,max-height as mandatory v8.0 onwards.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,v-tpg.txt          | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
index f33282bd46a8..4b2126a78a3f 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
@@ -7,6 +7,7 @@ Required properties:
 
     "xlnx,v-tpg-5.0" (TPG version 5.0)
     "xlnx,v-tpg-7.0" (TPG version 7.0)
+    "xlnx,v-tpg-8.0" (TPG version 8.0)
 
   TPG versions backward-compatible with previous versions should list all
   compatible versions in the newer to older order.
@@ -33,11 +34,13 @@ Optional properties:
   video timings.
 
 - reset-gpios: Specifier for a GPIO that assert TPG (AP_RST_N) reset.
-  This property is mandatory for TPG v7.0.
+  This property is mandatory for TPG v7.0 and above.
 
 - xlnx,max-height: Maximum number of lines.
+  This property is mandatory for TPG v8.0. Value ranges from 64 to 7760.
 
 - xlnx,max-width: Maximum number of pixels in a line.
+  This property is mandatory for TPG v8.0. Value ranges from 64 to 10328.
 
 The xlnx,vtc and timing-gpios properties are mandatory when the TPG is
 synthesized with two ports and forbidden when synthesized with one port.
-- 
2.31.1

